### DISCUSSION TITLE
Assignment Universal Calculator Tafe240153

### DISCUSSION THREAD

**Discussion by christian00117998** [christian00117998/Tafe240153#1] [General]
Hello, this will be the start of our discussion.

**Comment by christian00117998** [comment]
Hey Patrick good day,

**Comment by christian00117998** [comment]
Hey Patrick, before we dive into development, I wanted to check which feature you prefer to work on - the mortgage calculator or currency conversion calculator? Let me know your preference so we can allocate tasks accordingly. Looking forward to your response. Thanks!

**Reply by Donut02** [reply]
Hi Christian, I'm more inclined towards working on the currency conversion calculator. It seems like an interesting challenge, and I believe I can contribute effectively to its development. Let's proceed with that feature. Thanks for checking in!

**Comment by christian00117998** [comment]
Hey Patrick, thanks for letting me know your preference. Working on the currency calculator sounds like a plan. And I will be handling the work on mortgage calculator. As we develop our feature branches. Let us make sure that there are no errors in the program, and we should be able to plan things out properly to minimize the error upon pushing and merging out branches.

**Reply by Donut02** [reply]
Got it, thanks for the update! Let's focus on minimizing errors and testing thoroughly. Excited to collaborate on this project!

**Comment by christian00117998** [comment]
Just give me heads up and let us communicate thoroughly so that we will be able to know what status we in. Thank you

**Reply by Donut02** [reply]
Got it, I'll keep you updated and let's communicate regularly to stay on track. Thanks!

**Comment by Donut02** [comment]
Hi Christian, Should we include error handling in our code for the mortgage calculator? If so, how should we handle invalid user inputs?

**Reply by christian00117998** [reply]
I think it's important to include error handling to provide a better user experience. We could display a friendly error message for invalid inputs and prompt the user to correct them.

**Comment by Donut02** [comment]
Is there a preferred naming convention for variables and methods that we should follow in our C# code?

**Reply by christian00117998** [reply]
We should follow the naming conventions provided by Microsoft's C# coding guidelines. This will make our code more readable and maintainable for anyone who works on it in the future.

**Comment by Donut02** [comment]
Any specific UI design guidelines or suggestions for improving user experience?

**Reply by christian00117998** [reply]
Let's aim for a clean and modern UI design. Following our company's style guide for consistency would be a good idea. We can add tooltips or help icons for users needing guidance
